spec.html

```diff
@@ -12100,9 +12100,14 @@ <h1>InitializeHostDefinedRealm ( ): either a normal completion containing ~unuse
       1. If the host requires use of an exotic object to serve as _realm_'s global object, let _global_ be such an object created in a host-defined manner. Otherwise, let _global_ be *undefined*, indicating that an ordinary object should be created as the global object.
       1. If the host requires that the `this` binding in _realm_'s global scope return an object other than the global object, let _thisValue_ be such an object created in a host-defined manner. Otherwise, let _thisValue_ be *undefined*, indicating that _realm_'s global `this` binding should be the global object.
       1. Perform SetRealmGlobalObject(_realm_, _global_, _thisValue_).
-      1. Let _globalObj_ be ? SetDefaultGlobalBindings(_realm_).
+      1. Let _result_ be Completion(SetDefaultGlobalBindings(_realm_)).
+      1. If _result_ is an abrupt completion, then
+        1. Remove _newContext_ from the execution context stack.
+        1. Return ? _result_.
+      1. Let _globalObj_ be ? _result_.
```

Comment:
At this point, we know that `_globalObj_` isn't an abrupt completion, so `?` isn't appropriate.
```suggestion
      1. Let _globalObj_ be ! _result_.
```